compose.yaml

```diff
@@ -7,6 +7,7 @@ services:
       - ./container/:/var/lib/bowtie
     env_file: config.env
     environment:
+      SITE_ID:
```

Comment:
Guessing this is because you still have the export prefix on your env file... If you take the export off you can set your envrc to `dotenv config.env` and remove your sourcing